packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -141,18 +141,24 @@ export default function OrgNav({
         />
 
         <ul className={ulContainer}>
-          {paymentVariant && current?.paymentStatus && (
-            <li>
-              <Badge
-                className={css`
-                  margin-right: 25px;
-                `}
-                variant={paymentVariant}
-              >
-                {current.paymentStatus.toUpperCase()}
-              </Badge>
-            </li>
-          )}
+          {current &&
+            current.paymentStatus &&
+            paymentVariant &&
+            (admin ||
+              ['supsended', 'locked', 'admin suspended'].includes(
```

Comment:
typo, suspended

Reply:
Also it's not matching the values from mms.

Reply:
good catch on the typo... maybe i messed up but the enum i have for payment status is: 

const OrgPaymentLabel = {
  Embargoed: 'embargoed',
  EmbargoConfirmed: 'embargo confirmed',
  Ok: 'ok',
  Warning: 'warning',
  Suspended: 'suspended',
  Closing: 'closing',
  AdminSuspended: 'admin suspended',
  Dead: 'dead',
  Locked: 'locked',
  Closed: 'closed',
} as const;

Reply:
Yeah I'm not sure what's being passed in from the API tbh. I don't think we even are.

Regardless, here we should instead have it be `[OrgPaymentLabel.Suspended, ...etc]`. That's the value of using enums - reduces string errors!

Reply:
If I use the value of the enums, typescript says that "EMBARGOED" is not assignable to "SUSPENDED", "ADMIN_SUSPENDED" etc.